src/sql/base/browser/ui/taskbar/taskbar.ts

```diff
@@ -25,6 +25,10 @@ export interface ITaskbarContent {
 	element?: HTMLElement;
 }
 
+export interface ITaskbarOptions extends IToolBarOptions {
```

Comment:
Just a little cleanup here as well - this seems better as an option to the toolbar instead of its own parameter